### ISSUE TITLE
Strange large command line after maximize gvim

### ISSUE THREAD

**Issue by tamlok** [vim-ctrlspace/vim-ctrlspace#181]
1. Open gvim at its initial size (such as 800 * 600);
2. Toggle CtrlSpace;
3. Maximize gvim;
4. `<Esc>` to exit CtrlSpace. As a result, the command line will be very large, occupying half of the screen.

**Comment by cometsong** [comment]
I've reproduced this issue using MacVim, plus terminal vim and neovim.

For now, to reduce the cmdline size, you can use `set cmdheight=1` (or 2, 3, etc)

**Comment by Konfekt** [comment]
Yes, `:CtrlSpace` and closing the window right after starting Vim leaves the command line open.
